src/source.ts

```diff
@@ -127,80 +128,57 @@ export class Source {
   }
 
   /**
-   * Create or update a security identity. See [Adding a Single Security Identity](https://docs.coveo.com/en/167) and [Security Identity Models](https://docs.coveo.com/en/139).
-   * @param securityProviderId
-   * @param securityIdentity
-   * @returns
+   * @deprecated use `identity.createSecurityIdentity`
```

Comment:
Another way to do breaking changes is to leave old code working, but mark it as `@deprecated`.

AKA: Instead of removing the implementation, you change it to something like this: 

```
public createSecurityIdentity(
    _securityProviderId: string,
    _securityIdentity: SecurityIdentityModel
  ) {
       // call the new module/function/class
       // you can still respect the old contract.
       // optional: you warn the user in the console. At the very least they should see the code as striked out in their IDE because of the deprecated tag.
   }
```

Then you can merge it in a minor release.

Once you are ready to do a major breaking changes, you go ahead and remove all `@deprecated` code.